src/components/AddItem/components/ProductImg.jsx

```diff
@@ -0,0 +1,54 @@
+import { useRef, useState } from "react";
+import styles from "./styles/ProductImg.module.css";
+
+export default function ProductImg({ preview, setPreview }) {
+  const inputRef = useRef(null);
+
+  const handleImageChange = (e) => {
+    const file = e.target.files?.[0];
+
+    if (preview) {
+      alert("이미지는 한 개만 업로드할 수 있습니다.");
+      return;
+    }
+
+    if (file) {
+      const reader = new FileReader();
+      reader.onloadend = () => {
+        setPreview(reader.result);
+      };
+      reader.readAsDataURL(file);
+    }
+  };
+
+  const handleDelete = () => {
+    setPreview(null);
+  };
+
+  return (
+    <div className={styles.wrapper}>
+      <label className={styles.uploadBox}>
+        <div className={styles.textContainer}>
+          <span className={styles.plus}>＋</span>
+          <p className={styles.text}>이미지 등록</p>
+        </div>
+        <input
+          type="file"
+          accept="image/*"
+          ref={inputRef}
+          onChange={handleImageChange}
+          style={{ display: "none" }}
+        />
+      </label>
+
+      {preview && (
+        <div className={styles.imagePreviewBox}>
+          <button className={styles.deleteButton} onClick={handleDelete}>
```

Comment:
❗️ 수정요청
버튼 타입을 적지 않으시면 reset을 기본값으로 가지게 됩니다~ 늘 명시해주시면 좋습니다.
```suggestion
          <button type="button" className={styles.deleteButton} onClick={handleDelete}>
```